homeassistant/components/fritzbox/manifest.json

```diff
@@ -2,7 +2,13 @@
   "domain": "fritzbox",
   "name": "AVM FRITZ!Box",
   "documentation": "https://www.home-assistant.io/integrations/fritzbox",
-  "requirements": ["pyfritzhome==0.4.0"],
+  "requirements": ["pyfritzhome==0.4.2"],
+  "ssdp": [
+    {
+      "st": "urn:schemas-upnp-org:device:fritzbox:1"
+    }
+  ],
   "dependencies": [],
-  "codeowners": []
+  "codeowners": [],
```

Comment:
Do you want to add yourself as codeowner?

Reply:
Still unsure, soon I'll have to cope with 3 little kids...

Reply:
... and a wife. :D